src/demo/models/index.js

```diff
@@ -27,20 +28,15 @@ export const init = state => {
   }
 
   // Report a synthetic pageview to Google Analytics.
-  if (window.ga) {
-    const modeToPage = {
-      [Modes.Loading]: 'loading',
-      [Modes.Words]: 'words',
-      [Modes.Training]: 'training',
-      [Modes.Predicting]: 'predicting',
-      [Modes.Pond]: 'pond',
-      [Modes.Instructions]: 'instructions',
-      [Modes.IntermediateLoading]: 'intermediateLoading'
-    };
-
-    const syntheticPagePath =
-      window.location.pathname + '/' + modeToPage[state.currentMode];
-    window.ga('set', 'page', syntheticPagePath);
-    window.ga('send', 'pageview');
+  const modeToPage = {
+    [Modes.Loading]: 'loading',
+    [Modes.Words]: 'words',
+    [Modes.Training]: 'training',
+    [Modes.Pond]: 'pond',
+    [Modes.Instructions]: 'instructions'
+  };
+  const page = modeToPage[state.currentMode];
+  if (window.ga && page) {
```

Comment:
I think it might be good to hoist all checks for `window.ga` inside the function rather than worrying about checking for it here.